roles/dev_deploy/defaults/main/configs.yml

```diff
@@ -30,6 +37,15 @@ generated_configs:
 #  - name: beholder_web.yml
 #    dest: "{{ sources_root }}/repo-beholder"
 #    mode: "0655"
-  - name: vars.env
+  - sname: vars.env.j2
+    dname: vars.env
     dest: "{{ sources_root }}/albs-web-server"
     mode: "0655"
+  - sname: albs-sign-file.env.j2
+    dname: .env
+    dest: "{{ sources_root }}/albs-sign-file/"
+    mode: "0655"
+  - sname: alma-tests-cacher-vars.yaml.j2
+    dname: vars.yaml
```

Comment:
The `docker-compose` file in `albs-web-server` expects the environment file to be named `vars.env` instead of `vars.yaml`, which breaks the `docker-compose build`.

https://github.com/AlmaLinux/albs-web-server/blob/master/docker-compose.yml#L423-L424

This does contradict the `README` in the `alma-tests-cacher` repo, though: https://github.com/AlmaLinux/alma-tests-cacher#getting-started